recipes/orientdb/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "orientdb" %}
+#{% set version="2-2-29" %}
+{% set version="2.2.29" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+
+source:
+#  url: https://bit.ly/orientdb-ce-imps-{{ version }}-linux
+  url: https://bit.ly/orientdb-ce-imps-2-2-29-linux
+  sha256: ed6e65b18fed70ace3afa780a125100a19899e9b18f4d6e9bc1111e7ee88d752
+
+build:
+  number: 0
+  skip: true  # [win]
```

Comment:
How does this actually get built since there is no build.sh or command entry here?